src/main/java/com/sigma/cinematicketpro/filter/JwtAuthenticationFilter.java

```diff
@@ -0,0 +1,62 @@
+package com.sigma.cinematicketpro.filter;
+
+import com.sigma.cinematicketpro.service.AppUserService;
+import com.sigma.cinematicketpro.util.JwtTokenUtils;
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpHeaders;
+import org.springframework.lang.NonNull;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.web.authentication.WebAuthenticationDetailsSource;
+import org.springframework.stereotype.Component;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+import java.io.IOException;
+@Component
+@RequiredArgsConstructor
+public class JwtAuthenticationFilter extends OncePerRequestFilter {
```

Comment:
looks like it has to be a config instead + you just copy-pasted this block of code from somewhere

Reply:
fixed